rules/Arguments/ArgumentDefaultValueReplacer.php

```diff
@@ -31,13 +32,55 @@ public function processReplaces(
             if (! isset($node->params[$replaceArgumentDefaultValue->getPosition()])) {
                 return null;
             }
+            $this->processParams($node, $replaceArgumentDefaultValue);
```

Comment:
this will have new line after if, please run:

```bash
bin/rector process rules/Arguments/ArgumentDefaultValueReplacer.php
```

also on other files changed.

Reply:
Fixed